crab-dev.spec

```diff
@@ -3,7 +3,7 @@
 #For any other change, increment version_suffix
 ##########################################
 %define version_suffix 02
-%define crabclient_version v3.200531
+%define crabclient_version v3.200531.patch1
```

Comment:
why adding a patch suffix? can not we just increment the date? version checking scheme will fill to find the latest version  if version is properly followed

Reply:
I was curious if this could just work. I do not want it to become the latest version. It is effectively a dead branch for a limited scope (help one user our of troubles). I have already a latest tag in the standard numbering which have not queued for build yet becasue I was away. In the latest tag myproxy credential handling is different and this patch can't be appliet. I have to decide it to completely redo the myproxy delegation stuff or consider this an isolated incident and forget about it.

Reply:
anyhow myproxy.cern.ch admins solved the current problem already. So this patch may not be needed at all. I'd like to know if it could have worked (no problem in missing up crab-dev for a couple days in case). But almost certainly I will roll back the tag right after.

Reply:
rolling back